src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -9826,25 +9774,13 @@ namespace SurfaceGeometry {
             }
 
             // Warning if setpoint is unintentionally zero
-            if (windowShadingControl.SetPoint == 0 && ControlType != "ALWAYSON" && ControlType != "ALWAYSOFF" &&
-                ControlType != "ONIFSCHEDULEALLOWS" && ControlType != "SCHEDULE" && ControlType != "ONIFHIGHGLARE" && ControlType != "GLARE" &&
-                ControlType != "DAYLIGHTILLUMINANCE") {
```

Comment:
`"GLARE"` and `"DAYLIGHTILLUMINANCE"` don't point to valid IDD choices anymore and were removed from the `if` condition.